docs-release-notes.md

```diff
@@ -1,3 +1,11 @@
+# August 19, 2022
+
+## Manage & Deploy Apps
+- Secure: Add info about DNS Filtering when communicating from a Sync client to the Sync Server
+
+## Device Sync
+- Define Data Access Patterns/Sync Rules and Permissions: Restore collection-specific rule example and info
```

Comment:
```suggestion
- Define Data Access Patterns/Sync Rules and Permissions: Restore collection-specific rule examples and info
```
Did you mean 'example' or 'examples'?

Reply:
I actually did mean "example" in this case, but good question!